core/src/genesis.rs

```diff
@@ -243,6 +255,32 @@ impl GenesisTransaction {
     }
 }
 
+impl From<(&str, &str, &PublicKey)> for GenesisTransaction {
+    fn from((name, domain, pubkey): (&str, &str, &PublicKey)) -> Self {
+        Self {
+            isi: vec![
+                RegisterBox::new(IdentifiableBox::Domain(Domain::new(domain).into())).into(),
+                RegisterBox::new(IdentifiableBox::NewAccount(
+                    NewAccount::with_signatory(
+                        iroha_data_model::account::Id::new(name, domain),
+                        pubkey.clone(),
+                    )
+                    .into(),
+                ))
+                .into(),
+            ],
+        }
+    }
+}
```

Comment:
That is kinda strange implementation of from. Maybe you can create a named method for this conversion? Right now it can implicitly convert tuple of 3 elements to a transaction.